presto-main/src/test/java/com/facebook/presto/operator/aggregation/noisyaggregation/sketch/TestBitmap.java

```diff
@@ -29,12 +30,34 @@ public class TestBitmap
     private TestBitmap() {}
 
     @Test
-    public static void testRoundTrip()
+    public static void testRoundTripFullLength()
     {
         byte[] bytes = randomBytes(100);
```

Comment:
This shouldn't be random, at all. Instead fill with a known array of 100 bytes.

Reply:
This is deterministic now, but I'll go replace all the byte strings in this class with hardcoded values instead.